.github/workflows/model_jobs.yml

```diff
@@ -12,6 +12,12 @@ on:
       slice_id:
         required: true
         type: number
+      runner:
+        required: true
+        type: string
+      docker:
+        required: true
+        type: string
```

Comment:
just to allow using different runner and docker image.

(eventually, we probably move all runner to `ci` group and we no longer need `runner`)